integration_tests/snaps/simple-svn/snapcraft.yaml

```diff
@@ -0,0 +1,11 @@
+name: test-package
+version: 0.1
+summary: svn integration test
+description: use svn
+confinement: strict
+
+parts:
+    svn:
+        plugin: make
+        source: .
+        source-type: svn
```

Comment:
@elopio - I'm a little confused here. the git-* and hg-* directories have almost the same contents. What is different, and if applicable, what should I put in the make file and test file?

@chadmiller - Sort of going with the comment I left for elopio, would you suggest I do it as separate parts in this snapcraft.yaml, or in separate tests, and if so, would I rename the directory from simple-svn to svn-*?